### ISSUE TITLE
Email Verification Process is Buggy

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/Pilcrow#1968]
**Describe the bug**
When newly registering a user on Pilcrow, the banner that appears at the top of the page uses white text on the warning brand color resulting in a failing 1.69 contrast ratio. Verifying a user's email address does not make this banner disappear. The page must be first refreshed. The landing page for verifying an email address has a strange layout.

**To Reproduce**

1. Register a new user on Pilcrow
2. On the redirected page, notice that the banner has a contrast ratio that fails to meet accessibility standards
3. Go to MailHog (or whatever email inbox) and click the 'Verify Email Address' button within the Verify Email Address email
4. On the resulting landing page, notice that the banner still appears informing the user that their email address is still unverified
5. Notice that the alignment of elements on this landing page is off

**Expected behavior**
The banner should have passing color contrast. The banner should disappear immediately after the user verifies their email without having to refresh the page. The elements on the email verification landing page should have a strong and balanced alignment. 

**Screenshots**
[Screenshot of the email verification landing page showing that the unverified email banner still appears and features white text on the yellow background of the warning brand color. The email icon of the landing page does not align with the adjacent descriptive text. The 'Dashboard' link is shoved into this section further contributing to the alignment issues of this landing page.](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbVhsIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--d80130e5d2927bed22570175d45f9ec570bdaa0a/pilcrow.lndo.site_verify-email_1695867577_3bd25987acbd62826d88ab8d96e7859d727bffbb08bdb6c06a30138c35096a94%28iPad%20Air%29.png)
